website/content/en/status/report-2023-07-2023-09/desktop-convenience.adoc

```diff
@@ -0,0 +1,25 @@
+=== Filling gaps in the FreeBSD desktop experience
+
+Links: +
+link:https://github.com/outpaddling/qmediamanager[External media manager] URL: link:https://github.com/outpaddling/qmediamanager[] +
+link:https://github.com/outpaddling/devd-mount[devd-based automounter] URL: link:https://github.com/outpaddling/devd-mount[] +
+link:https://github.com/outpaddling/npmount[SUID mount tool] URL: link:https://github.com/outpaddling/npmount[] +
+link:https://github.com/outpaddling/freebsd-update-notify[Popup notification for updates] URL: link:https://github.com/outpaddling/freebsd-update-notify[]
+
+Contact: Jason Bacon <jwb@FreeBSD.org>
+
+The pkg:sysutils/desktop-installer[] port, available for over a decade now, quickly configures a bare FreeBSD system with any desktop environment or window manager.
+However, the FreeBSD base and ports collection has been missing some common features that end-users have come to expect from a desktop OS.
```

Comment:
```suggestion
However, the FreeBSD base and ports collection has been missing some common features that end users have come to expect from a desktop OS.
```
(My instinct tells me "end user" is a noun here and therefore needs no hyphen, but it's been known to be wrong.)